data/strings/strings.txt

```diff
@@ -12888,7 +12890,7 @@
     eu = Negozioa gehitu
     fa = اضافه کردن یک تجارت
     fi = Lisää organisaatio
-    fr = Ajouter une organisation
+    fr = Ajouter une entreprise
```

Comment:
Not sure that "entreprise"  is the best word here, organisation maybe better
I don't see where this string will appear in the app, what is the context?

Reply:
This translation is used on place page when you select a building, to create a new POI

Reply:
Ok I found it 👍
when i press on a building there is the string [placepage_add_business_button] en = Add business
when i long press anywhere else there is the string [placepage_add_place_button] en = Add a place to the map

both seems to open the editor in the same way, I don't see why we need two different strings, and moreover a building not necessarily contain a business (school, amenity, association etc...)